src/inmanta/parser/plyInmantaParser.py

```diff
@@ -799,20 +806,36 @@ def p_constants_collect(p: YaccProduction) -> None:
     p[0] = p[3]
 
 
-def p_param_list_collect(p: YaccProduction) -> None:
-    """param_list : ID '=' operand ',' param_list"""
-    p[5].insert(0, (p[1], p[3]))
-    p[0] = p[5]
+def p_wrapped_kwargs(p: YaccProduction) -> None:
+    "wrapped_kwargs : '*' '*' operand"
+    p[0] = WrappedKwargs(p[3])
 
 
-def p_param_list_term(p: YaccProduction) -> None:
-    "param_list : ID '=' operand"
-    p[0] = [(p[1], p[3])]
+def p_param_list_element_explicit(p: YaccProduction) -> None:
```

Comment:
it is perhaps a good idea to add a comment somewhere of the type of param_list  and what is put where.

Reply:
I pushed some changes. Could you check if they reflect what you mean?

Reply:
ok!